content/sensu-core/1.6/reference/clients.md

```diff
@@ -419,6 +420,18 @@ title="typically accurate within 500ms">consistency</abbr> between standalone
 checks _across an entire infrastructure_ (assuming that system clocks are
 synchronized via [NTP][23]).
 
+## Managing clients
+
+### Delete a client
+
+While shutting down a client stops all check and keepalive processes, it doesn't affect the history of the client or remove existing keepalive results.
```

Comment:
Wonder if for `check and keepalive processes` use `check results and keepalives` instead, as the process you are shutting down is the `sensu-client` here.

Reply:
Great callout! I've updated this.